smpe/bld/SMPMCS.txt

```diff
@@ -52,6 +52,9 @@
 ++PROGRAM(ZWELNCH)   SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
 ++PROGRAM(ZWESAUX)   SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
 ++PROGRAM(ZWESIS01)  SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
+++PROGRAM(ZWECFG31)  SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
+++PROGRAM(ZWECFG64)  SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
+++PROGRAM(ZWERXCFG)  SYSLIB(SZWEAUTH) DISTLIB(AZWEAUTH) RELFILE(3) .
```

Comment:
SZWELOAD is like the non-apf version of SZWEAUTH. Not sure how to add it here but I assume saying SZWEAUTH here isn't going to work.